scripts/pylib/twister/twisterlib.py

```diff
@@ -3163,13 +3163,17 @@ def load_quarantine(self, file):
         # platforms and scenarios for each entry in quarantine yaml
         quarantine_list = []
         for quar_dict in quarantine_yaml:
+            scenarios = quar_dict['scenarios']
+            scenarios = [s for s in scenarios if s is not None]
             if quar_dict['platforms'][0] == "all":
                 plat = self.platform_names
             else:
                 plat = quar_dict['platforms']
+                self.verify_platforms_existence(plat, f"quarantine-list")
```

Comment:
this function was invoked in other place too, but you add a new support for platform with none and only filter it in this place, you should handle those place too.
or I think you can filter none platforms firstly like line 3173 , then invoke verify function, in this case, don't need to modify verify_platforms_existence function.

Reply:
Originally, I added "support" for `null` platform in `verify_platforms_existence(...)` function at first to be able to properly handle situation in yaml file, when somebody would like to enforce no execution with `--integration` mode:
```
integration_platforms:
  - null
```
Then when I wanted to apply the same "handling" to verify platform names from quarantine file I observed that `null` platform is converted into None object which cannot be used with `join` function in 3175 line and cause error. That's why I added filtering of `null` platform in 3173 line.

I think that this is better approach, because user is still informed in debug message, that `null` platform is detected (in `verify_platforms_existence(...)` function) and in the same time it do not require to change parsing of each platform lists and removing every `null` platform (only in quarantine case).